chromium_src/third_party/blink/renderer/core/events/mouse_event.h

```diff
@@ -0,0 +1,48 @@
+/* Copyright (c) 2022 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_CHROMIUM_SRC_THIRD_PARTY_BLINK_RENDERER_CORE_EVENTS_MOUSE_EVENT_H_
+#define BRAVE_CHROMIUM_SRC_THIRD_PARTY_BLINK_RENDERER_CORE_EVENTS_MOUSE_EVENT_H_
+
+#include "third_party/blink/renderer/core/frame/local_dom_window.h"
+
+#define screenX                                                            \
+  screenX() const {                                                        \
+    auto* local_dom_window = DynamicTo<LocalDOMWindow>(view());            \
```

Comment:
this also doesn't appear to be checking the setting value? Also in general I think we need a lot more comments explaining what we're doing in these. Links to docs are ok

Reply:
actually I'm confused by what's going on here. Why wouldn't LocalDOMWindow do these calculations internally (screenX - screenX_ChromiumImpl)?

Reply:
I'm using the helper function here now (see below).